HIRS_Utils/src/main/java/hirs/tpm/eventlog/uefi/UefiGuid.java

```diff
@@ -0,0 +1,415 @@
+ case "a210f973-229d-4f4d-aa37-9895e6c9eaba":reference = "DpcDxe"; break;
+ case "025bbfc7-e6a9-4b8b-82ad-6815a1aeaf4a":
+                reference = "MNP Network Service Driver - MnpDxe"; break;
+ case "b44b2005-42bc-41c9-80af-abd7dc7d6923":reference = "RSTesSATAEFI"; break;
+ case "15e1e31a-9f9d-4c84-82fb-1a707fc0f63b":reference = "RSTeSATAEFI"; break;
+ case "2cc25173-bd9f-4c89-89cc-29256a3fd9c3":reference = "RSTesSATALegacy"; break;
+ case "bd5d4ca5-674f-4584-8cf9-ce4ea1f54dd1":reference = "RSTeSATALegacy"; break;
+
+ //  WinNt GUIDs, add if they are still found in use
+ //https://sourceforge.net/p/uefinotes/wiki/FV%20Sources/?version=3
+ case "fc5c7020-1a48-4198-9be2-ead5abc8cf2f":reference = "BdsDxe"; break;
+ case "d0893f05-b06d-4161-b947-9be9b85ac3a1":reference = "SnpNt32Dxe"; break;
+ case "9b3ada4f-ae56-4c24-8Dea-f03b7558ae50":reference = "PcdPeim"; break;
+ case "34c8c28F-b61c-45a2-8f2e-89e46becc63b":reference = "PeiVariable"; break;
+ case "fe5cea76-4f72-49e8-986f-2cd899dffe5d":reference = "FaultTolerantWriteDxe"; break;
+
+ // Linux Boot Image files
+ // UEFI Platform Initialization (PI) specifications
+ // Driver Execution Environment (DXE) Architectural protocols and platform modules
+ //https://github.com/linuxboot/linuxboot/blob/master/boards/winterfell/image-files.txt
+ case "5ae3f37e-4eae-41ae-8240-35465b5e81eb":reference = "CORE_DXE"; break;
+ case "cbc59c4a-383a-41eb-a8ee-4498aea567e4":reference = "DXE Runtime"; break;
+ case "3c1de39f-d207-408a-aacc-731cfb7f1dd7":reference = "DXE PciBus"; break;
+ case "80e66e0a-ccd1-43fa-a7b1-2d5ee0f13910":reference = "DXE PciRootBridge"; break;
+ case "9f3a0016-ae55-4288-829d-d22fd344c347":reference = "DXE AmiBoardInfo"; break;
+ case "13ac6dd0-73d0-11d4-b06b-00aa00bd6de7":reference = "DXE EBC"; break;
+ case "e03abadf-e536-4e88-b3a0-b77f78eb34fe":reference = "CPU DXE"; break;
+ case "b7d19491-e55a-470d-8508-85a5dfa41974":reference = "SBDXE"; break;
+ case "e23f86e1-056e-4888-b685-cfcd67c179d4":reference = "DXE SBRun"; break;
+ case "e4ecd0b2-e277-4f2b-becb-e4d75c9a812e":reference = "NBDXE"; break;
+ case "5ad34ba6-f024-2149-52e4-da0398e2bb9" :reference = "DXE Services Table"; break;
+ // ACPI configuration and tables
+ case "750890a6-7acf-4f4f-81bd-b400c2bea95a":reference = "AcpiModeEnable"; break;
+ case "d4c05cd1-5eae-431d-a095-13a9e5822045":reference = "MPST"; break;
+ case "db93cb2c-bf1c-431a-abc8-8737bc2afc1f":reference = "PRAD-ACPI-table"; break;
+ case "3bc5b795-a4e0-4d56-9321-316d18a7aefe":reference = "PRAD"; break;
+ case "16d0a23e-c09c-407d-a14a-ad058fdd0ca1":reference = "ACPI"; break;
+ case "26a2481e-4424-46a2-9943-cc4039ead8f8":reference = "S3Save"; break;
+ case "efd652cc-0e99-40f0-96c0-e08c089070fc":reference = "S3Restore"; break;
+ case "8c783970-f02a-4a4d-af09-8797a51eec8d":reference = "PowerManagement"; break;
+ case "299141bb-211a-48a5-92c0-6f9a0a3a006e0":reference = "PowerManagement-ACPI-table"; break;
```

Comment:
This has a different length. Is this the correct full ID?

Reply:
Extra 0 at the end.